src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Card.java

```diff
@@ -7,11 +7,14 @@
 import javax.persistence.*;
 
 @Entity
+@Table(uniqueConstraints = {@UniqueConstraint(columnNames = {"hand_id", "round_id"})})
```

Comment:
For later: is isnt it possible to use the java property names?

Reply:
You mean hand and round? No unfortunately not.